python/mxnet/_ffi/_cython/object.pxi

```diff
@@ -47,6 +51,29 @@ cdef inline object make_ret_object(void* chandle):
     (<ObjectBase>obj).chandle = chandle
     return obj
 
+class PyNativeObject:
+    """Base class of all MXNet objects that also subclass python's builtin types."""
```

Comment:
Which objects are that?

Reply:
String object in https://github.com/apache/incubator-mxnet/pull/19685/files#diff-a462320ba1e5c8161fe0f15fb415bcb794f251a845dd33e61675b97944a846c6R95-R120